generator-foodq/generators/app/index.js

```diff
@@ -353,9 +354,8 @@ module.exports = class extends Generator {
 	}
 
 	writing() {
-		if (_.get(this.log, "showProgress")) {
-			this.log.showProgress("FoodQ is generating.");
-		}
+		this.appWizard.messages.showProgress("FoodQ is generating.");
```

Comment:
wouldn't it be more clear --> this.appWizard.showProgress("FoodQ is generating.");

Reply:
yes, but in the future you will have commands and other objects
now it looks similar to vscode convention

Reply:
ok